packages/core/core-flows/src/cart/workflows/complete-cart.ts

```diff
@@ -136,6 +138,10 @@ export const completeCartWorkflow = createWorkflow(
 
       const paymentSessions = validateCartPaymentsStep({ cart })
 
+      beforePaymentAuthorization = createHook("beforePaymentAuthorization", {
+        input,
+      })
```

Comment:
question: What role with this hook play? We typically try to design the "API" of the hook with a specific purpose in mind, so maybe we can do the same here.

I have the same question for all other hooks. Could you maybe add this to the PR description?